cmake/Installation.cmake

```diff
@@ -32,9 +32,20 @@ if(APPLE)
   " COMPONENT Runtime
   )
 else()
+  include( GNUInstallDirs )
+  # follow the xdg-desktop specification
+  set(QTOX_DESKTOP_DIR "${CMAKE_INSTALL_FULL_DATAROOTDIR}/applications")
+  set(QTOX_DESKTOP_FILE "io.github.qtox.qTox.desktop")
   install(TARGETS ${PROJECT_NAME} RUNTIME DESTINATION "bin")
-  install(FILES "res/qTox.appdata.xml" DESTINATION "share/metainfo")
-  install(FILES "qtox.desktop" DESTINATION "share/applications")
+  install(FILES "res/io.github.qtox.qTox.appdata.xml" DESTINATION "${CMAKE_INSTALL_FULL_DATAROOTDIR}/metainfo")
```

Comment:
Along the same lines using `appstream-util validate-relax` during `test` is very useful.